modules/end-user-guide/partials/con_jetbrains-ides.adoc

```diff
@@ -0,0 +1,27 @@
+[id="con_jetbrains-ides_{context}"]
+= Support JetBrains IDEs
+
+In this section you'll find information about supported JetBrains IDEs
+that can be used in Eclipse Che workspace.
```

Comment:
Hello, FYI due to whitelabelling, Eclipse Che should not be used
`{prod}` should be used instead

the same for `Kubernetes` etc.

You may take a look at all attributes there: https://github.com/eclipse/che-docs/blob/master/antora-playbook.yml#L33-L112

Reply:
Fixed. Thanks for pointing it.